jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -313,6 +313,8 @@
             # verify-godeps compares against upstream, but remote/master might be stale
             # if .git was retained between runs. Update it explicitly here.
             git fetch remote master:refs/remotes/remote/master
+            # similarly, verify-munge-docs compares against the latest release branch.
+            git fetch remote release-1.4:remote/release-1.4
```

Comment:
this seems unmaintainable in the long term.

can we depend on something in the kubernetes repo to tell us what branch is the latest release branch?